src/app/app.module.ts

```diff
@@ -1,18 +1,33 @@
 import { NgModule } from "@angular/core";
 import { BrowserModule } from "@angular/platform-browser";
-
 import { AppRoutingModule } from "./app-routing.module";
 import { AppComponent } from "./app.component";
+import { FormsModule } from "@angular/forms";
+import { DialogModule } from "primeng/dialog";
+import { ReactiveFormsModule } from "@angular/forms";
+
+import { ButtonModule } from "primeng/button";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+
 import { HomePageComponent } from "./home-page/home-page.component";
+import { TaskFormComponent } from "./task-form/task-form.component";
+
+
```

Comment:
i think lines 8, 14, 15 should be deleted

Reply:
better now, ty